Documentation/Bookmarks/README.md

```diff
@@ -36,7 +36,7 @@ Bookmarks:
     /// Creates a `Bookmarks` component.
     /// - Parameters:
     ///   - isPresented: Determines if the bookmarks list is presented.
-    ///   - mapOrScene: A `GeoModel` authored with pre-existing bookmarks.
+    ///   - geoModel: A `GeoModel` authored with pre-existing bookmarks.
```

Comment:
Basemap gallery [uses this name](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/blob/1e04a58687c7ff5a4d2e7f7fa29413bf8c993653/Sources/ArcGISToolkit/Components/BasemapGallery/BasemapGallery.swift#L55) for a parameter of the same type.